packages/office-addin-lint/test/src/test.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as assert from "assert";
+import * as fs from "fs";
+import * as mocha from "mocha";
+import * as path from "path";
+import * as prettierLint from "../../src/lint";
+
+describe("test cases", function() {
+    it("eslint and prettier commands", async function() {
+        const inputFile = "./test/cases/basic/functions.ts";
+        const lintExpectedCommand = "eslint ./test/cases/basic/functions.ts";
+        const lintFixExpectedCommand = "eslint --fix ./test/cases/basic/functions.ts";
+        const prettierExpectedCommand = "prettier --parser typescript --write ./test/cases/basic/functions.ts";
+        const lintCheckCommand = prettierLint.lintCheckCommand(inputFile);
+        assert.strictEqual(lintCheckCommand, lintExpectedCommand, "eslint lint command");
```

Comment:
Can we have a message saying what is wrong instead of "eslint lint command". Suggest - "Lint command does not match expected value" or use the default message which will list out both values with AssertionError.

Reply:
updated